kolibri/utils/server.py

```diff
@@ -100,7 +107,7 @@ def start(port=8080, run_cherrypy=True):
     :param: port: Port number (default: 8080)
     """
 
-    run_services()
+    run_services(port=port)
```

Comment:
Perhaps, yeah -- sometimes I like to be explicit with named arguments, as it makes reading the function call clearer. In this case it doesn't add much value since the variable has the same name as the argument.